src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -633,6 +648,41 @@ class spell_warr_victory_rush : public SpellScript
     }
 };
 
+// 262161 - Warbreaker
+class spell_warr_warbreaker : public SpellScript
```

Comment:
the code seems identical to `spell_warr_colossus_smash`, you can assign the spellid 262161 to `spell_warr_colossus_smash` aswell (so both have the same ScriptName)